File: JavaBackEnd/test/src/main/java/com/yohane/test/dao/Data/ResponseCode/Result.java
package com.yohane.test.dao.Data.ResponseCode;

import com.sun.xml.internal.ws.developer.Serialization;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.io.Serializable;

/**
 * Created on 2023/4/13
 *
 * @Time 9:56
 * @Program test
 * @Author YoHaNe/XiaoShuai
 */
@Data
@AllArgsConstructor
public class Result implements Serializable {

    private static final long serialVersionUID = -1160305195842100549L;

    private Integer code;

    private String message;

    private Object data;

}

File: Management/MusicManagement/MusicManagement/Service/DAO/Result.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Service.DAO
{
    public class Result
    {
        private int Code;

        private string Message;

        private object Data;

        public int code { get => Code; set => Code = value; }
        public string message { get => Message; set => Message = value; }
        public object data { get => Data; set => Data = value; }
    }
}
